package.json

```diff
@@ -81,29 +81,28 @@
     "ui:build:package": "cpy ./src/clr-ui/package.json ./src/clr-ui/README.md ./dist/clr-ui/; replace '@VERSION' $npm_package_version ./dist/clr-ui/package.json",
     "ui:lint": "stylelint --config ./src/clr-core/.stylelintrc \"src/clr-ui/**/*.scss\"",
     "ui:lint:fix": "stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-ui/**/*.scss\"",
-    "core:start": "npm-run-all core:build -p core:build:sass:watch 'core:build:ts -- -w' 'ng serve dev-core'",
-    "core:dev:build": "ng build dev-core --aot --sourceMap=false",
-    "core:dev:build:prod": "ng build dev-core --prod",
-    "core:lint": "npm run prelint && tslint -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" -e \"src/clr-core/**/*.css.ts\" && tslint -p . -c ./src/dev-core/tslint.json \"src/dev-core/**/*.ts\" && stylelint --config ./src/clr-core/.stylelintrc \"src/clr-core/**/*.scss\" \"src/dev-core/**/*.scss\"",
-    "core:lint:fix": "npm run prelint && tslint --fix -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" && tslint --fix -p . -c ./src/dev-core/tslint.json \"src/dev-core/**/*.ts\" && stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-core/**/*.scss\" \"src/dev-core/**/*.scss\"",
+    "core:start": "npm-run-all core:build -p core:build:watch storybook",
+    "core:start:old": "npm-run-all core:build -p core:build:watch 'ng serve dev-core'",
```

Comment:
will `core:start:old` hang around for a while? or did you mean to delete it before the merge?

Reply:
I was planning on leaving it until I can open a separate PR to remove the dev app. It wont build it during the CI build but still lets us run it in dev mode. I didnt want to change more than I had to in a single PR.